index.html

```diff
@@ -0,0 +1,150 @@
+                    <h2>Celebrities</h2>
+                    <h3>Famous musicians have going to participate in events all the days. Dont lose this oportunity!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/theme-park-icon.png" alt="" class="card-img">
+                    <h2>Theme Park</h2>
+                    <h3>You can visit diverse atractions all the day!Dont fogot go to the mountain russian!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/sun-icon.png" alt="" class="card-img">
+                    <h2>Non stop!</h2>
+                    <h3>This party dont gonna stop never! so you can be parting 24/7</h3>
+                </div>
+            </div>
+            <div class="button-container">
+            <button class="button-cards" id="join">Join to the party!</button>
+            </div>
+            <div class="grid-container-desktop">
+                <div class="card"><img src="sources/mobile-version/images/mask-icon.png" alt="" class="card-img">
+                    <h2>thematic events</h2>
+                    <h3>Feel free to play as a character in the night of the cartoons, and roleplay in the other special events!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/tickets-icon.png" alt="" class="card-img">
+                    <h2>Free Bar & Entry</h2>
+                    <h3>You can ask for a free drink every day. As well get a free entry to one show in the last day of the event</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/superstar-icon.png" alt="" class="card-img">
+                    <h2>Celebrities</h2>
+                    <h3>Famous musicians have going to participate in events all the days. Dont lose this oportunity!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/theme-park-icon.png" alt="" class="card-img">
+                    <h2>Theme Park</h2>
+                    <h3>You can visit diverse atractions all the day!Dont fogot go to the mountain russian!</h3>
+                </div>
+                <div class="card"><img src="sources/mobile-version/images/sun-icon.png" alt="" class="card-img">
+                    <h2>Non stop!</h2>
+                    <h3>This party dont gonna stop never! so you can be parting 24/7</h3>
+                </div>
+            </div>
+            <div class="button-container">
+            </div>
+            <a href="" class="link-desktop">SEE THE WHOLE PROGRAM</a>
+        </section>
+        <section class="artists">
+            <div></div>
+            <h2>Featured Artists</h2>
+            <div class="tiny-line"></div>
+            <div class="container-speakers">
+            </div>
+            <div class="hidden-verification"></div>
+            <button class="button-artists">
```

Comment:
Greeting! You did a great work so far, but there are some changes that you should change to make your design match the requirement. 
According to the Design I suggest you stick to the design as much as possible.

> ## The Original Design
>
><img width="1440" alt="Screenshot 2023-07-01 at 1 41 54 AM" src="https://github.com/SaveryIV/First-Capstone-Project/assets/91237525/54ba9d02-84b9-4d50-a585-e713199a1c55">
>
> # Your Design
>
><img width="1440" alt="Screenshot 2023-07-01 at 1 43 49 AM" src="https://github.com/SaveryIV/First-Capstone-Project/assets/91237525/1b0a7585-1d5d-4b1a-a5b8-da725f3a8349">

Reply:
Hello @SaveryIV, you did well with your project but you seem to misunderstand the comment by the previous reviewer, for each speaker in the card, there is some spacing between them while yours looks clustered together. Please fix this review so you'd have a better homepage, I'd do well to highlight the spacing I'm referiing to so you'd take note. Kindly make sure to fix it before resubmitting for another review 😄 

Your design :
![image](https://github.com/SaveryIV/First-Capstone-Project/assets/20537712/6d261000-08e4-4267-a0e0-5291421f12b0)

Behance design :
![image](https://github.com/SaveryIV/First-Capstone-Project/assets/20537712/b5629b7c-935d-4dce-bbbe-43d97a619a0e)